packages/neuron-ui/src/locales/en.json

```diff
@@ -120,8 +122,12 @@
       "actions": {
         "broadcast": "Broadcast",
         "sign": "Sign and export",
-        "cancel": "Cancel"
-      }
+        "cancel": "Cancel",
+        "view-in-explorer": "View in Explorer"
+      },
+      "broadcast-transaction": "Broadcast Transaction",
+      "import-unsigned-transaction": "Import an unsigned transaction",
```

Comment:
I would suggest informing users with `Import a transaction to sign` because it delivers the required action and the goal. cc @Danie0918 @Sven-TBD